src/Targets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -16,18 +16,30 @@
     <LanguageServiceName Condition="'$(LanguageServiceName)' == ''">Visual Basic</LanguageServiceName>
   </PropertyGroup>
 
-  <!-- Targets -->
-
-  <!-- Returns Vbc command-line arguments for the language service -->
-  <Target Name="CompileDesignTime"
-          Returns="@(_CompilerCommandLineArgs)"
-          DependsOnTargets="_CheckCompileDesignTimePrerequisite;Compile"
-          >
+    <ItemGroup>
+     <_CompilerCommandLineArgs Include="@(VbcCommandLineArgs)"/>
+    </ItemGroup>
 
     <ItemGroup>
-      <_CompilerCommandLineArgs Include="@(VbcCommandLineArgs)"/>
+     <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)VisualBasic.ProjectItemsSchema.xaml;"/>
```

Comment:
Looking at Microsoft.VisualBasic.CurrentVersion.targets , it does set a context for ProjectItemsSchema , same for cSharp too.